packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -221,7 +221,7 @@ class VerboseLogger extends Logger {
     stopwatch.reset();
 
     String prefix;
-    const int prefixWidth = 12;
+    const int prefixWidth = 8;
```

Comment:
Ah, I was leaving space for 9999 ms. Items longer then that would have different indent than the rest of the verbose lines, but that seemed fine, given that those items are outliers.